server/src/main/java/org/elasticsearch/transport/Transports.java

```diff
@@ -36,7 +36,7 @@
      * used in assertions to make sure that we do not call blocking code from
      * networking threads.
      */
-    public static final boolean isTransportThread(Thread t) {
+    public static boolean isTransportThread(Thread t) {
```

Comment:
That one is interesting, I thought it would covered already by redundant modifiers? I guess this is a bug fix in checkstyle?

Reply:
It's a bug fix in checkstyle, reported in (https://github.com/checkstyle/checkstyle/issues/2068)